expression-language/4.0/_index.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: "Jakarta Expression Language 4.0 (milestone)"
-date: 2020-06-12
+title: "Jakarta Expression Language 4.0"
+date: 2020-07-16
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Expression Language defines an expression language for Java applications.
 
 * [Jakarta Expression Language 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.el/releases/4.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Expression Language 4.0 Specification Document](./expression-langauge-spec-4.0-RC2.pdf) (PDF)
-* [Jakarta Expression Language 4.0 Specification Document](./expression-langauge-spec-4.0-RC2.html) (HTML)
+* [Jakarta Expression Language 4.0 Specification Document](./expression-langauge-spec-4.0.pdf) (PDF)
+* [Jakarta Expression Language 4.0 Specification Document](./expression-langauge-spec-4.0.html) (HTML)
 * [Jakarta Expression Language 4.0 Javadoc](./apidocs)
 * [Jakarta Expression Language 4.0 TCK](http://download.eclipse.org/ee4j/jakartaee-tck/master/nightly/expression-language-tck-4.0.0.zip)([sig](),[sha](),[pub]())
```

Comment:
FYI, here's the Issue for getting this TCK staging area setup and processed...
https://github.com/eclipse-ee4j/jakartaee-tck/issues/360

Reply:
https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-900/eclipse-expression-language-tck-4.0.0.zip is now available.

Reply:
Just to confirm, that download directory contains the proposed final TCKs and (assuming no bugs) they won't change?

Reply:
Hmm, I will look into the "eclipse-" prefix in the file name.

Reply:
@markt-asf Do you expect that only final (passing 100%) TCKs would be in the staged folder?  I expect that we will keep making changes (e.g bug fixes + doc updates).  

When we have TCKs that are expected to be final, we will promote them to https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/promoted folder.

Also the file names look wrong with the "eclipse-" prefix, so I expect we will generate a "expression-language-tck-4.0.0.zip" instead (https://github.com/eclipse-ee4j/jakartaee-tck/issues/363 is for addressing the unexpected "eclipse-" prefix).

Reply:
I guess the "eclipse-" prefix is expected in [build_standalone-tcks.sh#L244](https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/docker/build_standalone-tcks.sh#L244)

Reply:
Based on the discussion in https://github.com/eclipse-ee4j/jakartaee-tck/issues/363 the eclipse-* bundles will be renamed to jakarta-*.

Also we require to update the userguides (especially the versions) for all the TCKs along with other possible changes. 

As Scott mentioned the https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-900 will contain the nightly EFTL bundles. We will move them to https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/promoted which can be used to run the certification tests.

Reply:
> Based on the discussion in [eclipse-ee4j/jakartaee-tck#363](https://github.com/eclipse-ee4j/jakartaee-tck/issues/363) the eclipse-* bundles will be renamed to jakarta-*.

Correct.  The final name should begin with `jakarta-`, but early versions in Jakarta EE 8 accidentally had the `eclipse-` prefix. Instead of forcing a change in all of the PRs, we decided to go ahead and change the name when we run the promotion script.  Going forward, we should try to use the `jakarta-` prefix.

> 
> Also we require to update the userguides (especially the versions) for all the TCKs along with other possible changes.

Correct.

Reply:
1. https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-900/ now has the bundles with "jakarta-" prefix. Once the TCKs are final they will be moved to https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/promoted.

2. I raised https://github.com/eclipse-ee4j/jakartaee-tck/pull/367  for updating the EL TCK userguide. I hope each API teams can pitch in to update the respective TCK userguides. 
Ref : https://github.com/jakartaee/specifications/pull/229#issuecomment-660733738

Reply:
This conversation looks to be resolved.